databases/relational-databases.md

```diff
@@ -292,14 +188,27 @@ Solve the quizzes mentioned below
 
 - What do you mean by Relational database?
 - What is CRUD in SQL?
-- What is DDL?
 - How does SELECT command work?
-- How does JOIN work?
-- What are the advantages of a Relational database?
-- What is an ORM?
-- How is PostgreSQL different from MySQL?
 - How do you model data for relational databases?
 
+### Common Mistakes & Misconceptions
+
+- A database can be referred to as a table or collection of tables, a database server, or SQL. You have to consider the context for better understanding.
+
+### Resources for Non-Relational Databases (OPTIONAL)
+
+- Different types of [non-relational databases](https://medium.com/@mark.rethana/introduction-to-nosql-databases-c5b43f3ca1cc) (non-sql databases):  
```

Comment:
Can you make this just normal text, not a bullet point and then un-indent the rest so it's just one-level list instead of two-level?